integration_tests/assets/docker-compose.documentation.override.yml

```diff
@@ -14,7 +14,7 @@ services:
       - ../../xivo_bus:/app/xivo_bus:ro
 
   spec-validator:
-    image: asyncapi/cli:latest
+    image: asyncapi/cli
```

Comment:
why remove the tag? Pretty sure no tag is equivalent to `latest`. You can also pin the version to the current one(`asyncapi/cli:1.2.34`)

Reply:
Readded the latest tag for now